### PR TITLE
Update Issue templates

### PR DESCRIPTION
**Pull request by goplayoutside3**
## Linked Issue and/or Talk Post

This is a follow up to the FEM Documentation discussion session at ZTM.

## Describe your changes

Update the text in both the bug report and new feature Issue templates. I also deleted the duplicate new feature Issue template.

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General
- [ ] Tests are passing locally and on Github
- [ ] Documentation is up to date and changelog has been updated if appropriate
- [ ] You can `yarn panic && yarn bootstrap` or `docker-compose up --build` and FEM works as expected

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ assignees: ''
 ---
 
 ## Package
+_Choose from the list: app-content-pages, app-content, lib-classifier, lib-panoptes-js, lib-react-components, unknown_
```
Comment (eatyourgreens) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Maybe make this a list with checkboxes?

```suggestion
_Choose from the list:_ 
- [ ] app-content-pages
- [ ] app-content
- [ ] lib-classifier
- [ ] lib-panoptes-js
- [ ] lib-react-components
- [ ] unknown
```

File: .github/ISSUE_TEMPLATE/feature-request--ux--ui--accessibility--design--etc.md [hunk 2]
```diff
@@ -1,21 +1,19 @@
...
 name: 'Feature request: UX, UI, accessibility, design, etc'
-about: Suggest an idea for this project related to UX, UI, accessibility, or design
+about: Suggest an idea or document a planned feature to-do
 title: ''
 labels: enhancement
 
 ---
 
 ## Package
+_Choose from the list: app-content-pages, app-content, lib-classifier, lib-panoptes-js, lib-react-components, unknown_
```
Comment (eatyourgreens) [inline .github/ISSUE_TEMPLATE/feature-request--ux--ui--accessibility--design--etc.md]:
```suggestion
_Choose from the list:_ 
- [ ] app-content-pages
- [ ] app-content
- [ ] lib-classifier
- [ ] lib-panoptes-js
- [ ] lib-react-components
- [ ] unknown
```

**Review by eatyourgreens — APPROVED** [review]
Looks good to me. I'd make the package list into a TODO list with checkboxes, so that people can tick the relevant package.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (goplayoutside3) [reply .github/ISSUE_TEMPLATE/bug_report.md]: oh! Good idea

**Pull request merged**